src/api-client.ts

```diff
@@ -149,10 +152,17 @@ export class RevAiApiClient {
     }
 
     async getTranscriptObject(id: string): Promise<RevAiApiTranscript> {
+        const stream = await this.getTranscriptObjectStream(id);
+        return stream.read();
+    }
+
+    async getTranscriptObjectStream(id: string): Promise<Readable> {
```

Comment:
Looks good to me. Let's discuss the longer term plan for defining output at the friday meeting. Example that we discussed earlier: should we have a singular function `client.getOutput(jobId, type, options)`?